config/sources/amd64.conf

```diff
@@ -0,0 +1,13 @@
+export INITRD_ARCH=amd64                # Dunno what this does.
```

Comment:
https://github.com/armbian/build/blob/master/packages/bsp/common/etc/initramfs/post-update.d/99-uboot#L7

Reply:
Thanks! fixed the comment.